dbt_utils.py

```
@@ -49,34 +93,42 @@ def fetch_schema_from_sql(sql: str, dialect: str):
     except Exception as e:
         raise Exception(str(e))
 
-def validate_whether_sql_has_columns(sql: str, dialect: str):
+
+def project_healthcheck(
```

Pylint (3.3.7)

[refactor] Too many arguments (7/5)

(R0913)